systemtest/src/main/java/io/strimzi/systemtest/Install.java

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.fabric8.kubernetes.api.model.rbac.ClusterRoleBinding;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.resources.kubernetes.ClusterRoleBindingResource;
+import io.strimzi.systemtest.resources.kubernetes.NetworkPolicyResource;
+import io.strimzi.systemtest.resources.kubernetes.RoleBindingResource;
+import io.strimzi.systemtest.resources.operator.BundleResource;
+import io.strimzi.systemtest.resources.specific.HelmResource;
+import io.strimzi.systemtest.resources.specific.OlmResource;
+import io.strimzi.systemtest.templates.kubernetes.ClusterRoleBindingTemplates;
+import io.strimzi.test.TestUtils;
+import io.strimzi.test.executor.Exec;
+import io.strimzi.test.k8s.KubeClusterResource;
+import io.strimzi.test.k8s.cluster.Minishift;
+import io.strimzi.test.k8s.cluster.OpenShift;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Stack;
+import java.util.stream.Collectors;
+
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+public class Install {
```

Comment:
```suggestion
public class Setup {
```

@im-konge @see-quick  wdyt?

Reply:
Echm...what about `InstallClusterOperator` or `SetupClusterOperator`...it will be more verbose and not only setup or install... )

Reply:
+1 for `SetupClusterOperator`

Reply:
`SetupClusterOperator` sounds good to me